<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-POM6KTNO" failures="2" tests="5" name="firsttestng" time="27.858" errors="0" timestamp="2023-08-05T12:30:34 IST">
  <testcase classname="com.tutorialsninja.qa.testcases.register" name="registerwithvalidcredentials" time="1.513"/>
  <testcase classname="com.tutorialsninja.qa.testcases.Login" name="loginwithvaildcredentials" time="0.948">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialsninja.qa.testcases.Login.loginwithvaildcredentials(Login.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginwithvaildcredentials -->
  <testcase classname="com.tutorialsninja.qa.testcases.Login" name="loginwithvaildcredentials" time="1.241">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialsninja.qa.testcases.Login.loginwithvaildcredentials(Login.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginwithvaildcredentials -->
  <testcase classname="com.tutorialsninja.qa.testcases.Login" name="Loginwithinvalidcredentials" time="0.611"/>
  <testcase classname="com.tutorialsninja.qa.testcases.search" name="searchcontext" time="0.555"/>
</testsuite> <!-- firsttestng -->
